[HOLD translations] Add existing owners error modal during bank account setup

jasperhuangg opened the pull request:
### Details
Previously, `setupWithdrawalAccount` would silently error when a user tried to add a withdrawal bank account that we already assigned an owner to. 

This PR adds some text similar to [the text in Web-Secure](https://github.com/Expensify/Web-Secure/blob/2f20e1b0b13b2d09081ecf0986a64a8af4a0a67f/site/app/dialogs/reimbursementAccount/existingCopiesWarning.jsx#L3-L49).

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/169896

### Tests/QA
1. Create a new workspace and click "Get Started".
2. Add a bank account manually following the steps [here](https://stackoverflow.com/c/expensify/questions/342).
3. In a different account, follow steps 1 and 2.
4. Verify that you're unable to add the account since the bank account already has an existing owner, and verify that the error step displays (see screenshots).

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [ ] iOS
- [x] Android

### Screenshots
_NOTE: I've since updated the header title to be capitalized [aka `Existing Owners`] (not shown in the screenshots below)_


#### Desktop
<img width="1203" alt="Screen Shot 2021-07-13 at 3 32 18 PM" src="https://user-images.githubusercontent.com/31285285/125412178-3e7dfe00-e3f1-11eb-86fb-87a780f70fd5.png">

#### Web
<img width="1792" alt="Screen Shot 2021-07-13 at 3 34 47 PM" src="https://user-images.githubusercontent.com/31285285/125412183-4047c180-e3f1-11eb-9445-f2a9963468ce.png">

#### Mobile-Web
<img width="374" alt="Screen Shot 2021-07-13 at 3 36 19 PM" src="https://user-images.githubusercontent.com/31285285/125412191-84b60644-75d5-4e37-862d-361420c7885f.png">

#### Android
![WechatIMG26](https://user-images.githubusercontent.com/31285285/125412562-9ddc0e00-e3f1-11eb-92b2-d1efa6567b31.jpeg)

cc @marcaaron Can you verify if the testing steps are correct? I was unable to test entirely on dev because I can't hit the Web-Secure API.

roryabraham committed:
Merge pull request #3908 from rdjuric/transitionFix

Improves transition when changing chats

(cherry picked from commit b1f09a3b276ea6b4771f285879d0ffa161f707ce)

jasperhuangg committed:
add todo

jasperhuangg commented on src/components/GrowlNotification/index.js:
```diff
@@ -27,33 +28,55 @@ const types = {
 };
 
-const INACTIVE_POSITION_Y = -255;
+const INACTIVE_POSITION_Y = -300;
```
Updated because the `BankAccountexistingOwners` component is taller than 255px.

jasperhuangg commented on src/components/GrowlNotification/index.js:
```diff
@@ -70,7 +93,7 @@ class GrowlNotification extends Component {
         Animated.spring(this.state.translateY, {
             toValue: val,
-            duration: 80,
+            duration: 100,
```
updated to account for the increase in `INACTIVE_POSITION_Y`

jasperhuangg commented on src/languages/en.js:
```diff
@@ -321,6 +321,13 @@ export default {
         checkHelpLine: 'Your routing number and account number can be found on a check for the account.',
         hasPhoneLoginError: 'To add a verified bank account please ensure your primary login is a valid email and try again. You can add your phone number as a secondary login.',
         hasBeenThrottledError: ({fromNow}) => `For security reasons, we're taking a break from bank account setup so you can double-check your company information. Please try again ${fromNow}. Sorry!`,
+        existingOwnersError: {
+            alreadyInUse: 'This bank account is already in use by ',
+            pleaseAskThemToShare: '.\n\nPlease ask them to share it with you.\n\n',
+            alternatively: 'Alternatively, you can ',
+            setUpThisAccountByYourself: 'set up this account by yourself',
+            validationProcessAgain: ' and go through the entire validation process again (may take up to a week).',
+        },
```
TODO: awaiting copy change

jasperhuangg commented on src/languages/es.js:
```diff
@@ -312,6 +312,15 @@ export default {
         checkHelpLine: 'Su número de ruta y número de cuenta se pueden encontrar en un cheque para la cuenta.',
         hasPhoneLoginError: 'Para agregar una cuenta bancaria verificada, asegúrese de que su inicio de sesión principal sea un correo electrónico válido y vuelva a intentarlo. Puede agregar su número de teléfono como inicio de sesión secundario.',
         hasBeenThrottledError: ({fromNow}) => `Por razones de seguridad, nos tomamos un descanso de la configuración de la cuenta bancaria para que pueda verificar la información de su empresa. Inténtalo de nuevo ${fromNow}. ¡Lo siento!`,
+
+        // TODO: Get translations after copy fix
+        existingOwnersError: {
+            alreadyInUse: 'This bank account is already in use by ',
+            pleaseAskThemToShare: '.\n\nPlease ask them to share it with you.\n\n',
+            alternatively: 'Alternatively, you can ',
+            setUpThisAccountByYourself: 'set up this account by yourself',
+            validationProcessAgain: ' and go through the entire validation process again (may take up to a week).',
+        },
```
TODO: awaiting copy change before asking for translations

rdjuric committed:
minor changes

OSBotify committed:
Merge pull request #3948 from Expensify/version-BUILD-2db53ecdca5e2762a5c394aef895123fb0f5f332

(cherry picked from commit 263ce68a2fc2df530ac88c047c71d4609dbc5ae0)

OSBotify committed:
Merge pull request #3949 from Expensify/cherry-pick-staging-3908

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #3956 from Expensify/update-staging-from-main

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #3962 from Expensify/update-staging-from-main

marcaaron committed:
navigate to concierge

marcaaron committed:
Add new copy + link to Concierge + Finish Setup

parasharrajat committed:
support to html pasting on web

parasharrajat committed:
pull latest changes

rdjuric committed:
removes unecessary comment

rushatgabhane committed:
make Button platform specific

rushatgabhane committed:
add pressOnEnter prop

rushatgabhane committed:
add eventListener on mount, rm on unmount

rushatgabhane committed:
define handleKeydown()

rushatgabhane committed:
press button on enter key

rushatgabhane committed:
eslint

rushatgabhane committed:
cleanup iouAmountPage by using pressOnEnter

rushatgabhane committed:
convert to class comp and use KeyboardShortcut

rushatgabhane committed:
rm platform specific code for Button

rushatgabhane committed:
eslint

johncschuster committed:
scrubbing Expensify Cash for New Expensify

rdjuric committed:
removes comment

parasharrajat committed:
fix: scrolling and layout

marcaaron committed:
translate

roryabraham committed:
Give disabled buttons the disabled cursor style

AndrewGable committed:
Merge pull request #3989 from Expensify/Rory-FixMobileInviteForm

Make button visible in WorkspaceInvitePage

(cherry picked from commit bb042f5810d4861abdc59b42177348d86233390f)

Commit:
Merge branch 'main' of https://github.com/Expensify/Expensify.cash into shawn-updateAppIcon

OSBotify committed:
Merge pull request #3990 from Expensify/version-BUILD-bb042f5810d4861abdc59b42177348d86233390f

(cherry picked from commit 5b90ea6638788d9f9b5fed52bef4961d17e07c11)

OSBotify committed:
Merge pull request #3991 from Expensify/cherry-pick-staging-3989

Commit:
changed one more image

marcaaron committed:
show loader when loading bank account

marcaaron committed:
fix conflict

thienlnam committed:
Merge pull request #3966 from Expensify/Rory-FixLineHeight

Only provide a default lineHeight when the fontSize is default

(cherry picked from commit d0c95782a407809b6a7b2f8872053f90aae00a16)

OSBotify committed:
Merge pull request #3996 from Expensify/version-BUILD-d0c95782a407809b6a7b2f8872053f90aae00a16

(cherry picked from commit 5964a2b602fef95fe67ca4d20862e6bbcf8d8908)

OSBotify committed:
Merge pull request #3997 from Expensify/cherry-pick-staging-3966

mountiny committed:
TEST - issue lifecycle

rdjuric committed:
Merge branch 'main' into fixTooltipAndDatePos

marcaaron committed:
fix navigation issue

jasperhuangg committed:
revert unneeded changes

jasperhuangg committed:
Merge branch 'main' of github.com:Expensify/Expensify.cash into jasper-existingOwnersCopy

jasperhuangg committed:
setup as modal step

jasperhuangg committed:
reorganize imports

jasperhuangg committed:
add missing page

jasperhuangg committed:
capitalize header titles

jasperhuangg commented:
@Julesssss This is held on some translations, but should be good for a review in the meantime, thanks!

rushatgabhane committed:
rm unnecessary interaction manager

mountiny committed:
Merge pull request #3998 from Expensify/vit-testingECashIssueLifecycle1

[TEST] - Test E.cash issue lifecycle webhook for employees

vitHoracek committed:
[TEST] Testing E.cash issue lifecycle - Delete test.md

rdjuric committed:
changes to pressable

rdjuric committed:
lint fixes

mountiny committed:
Merge pull request #4001 from vitHoracek/vit-testWebhook

[TEST] Testing E.cash issue lifecycle - Delete test.md

mountiny committed:
TEST - testing the issue lifecycle

parasharrajat committed:
Pulled changes

johncschuster committed:
updates from Tom's feedback

madmax330 committed:
Merge pull request #4003 from Expensify/vitTestingECashIssueLifecycle2

[TEST] - Test E.cash issue lifecycle webhook for employees 2

rdjuric committed:
restores componentDidUpdate lifecycle

Julesssss commented on src/languages/es.js:
```diff
@@ -371,6 +371,16 @@ export default {
         },
     },
+
+    // TODO: Waiting translations
```
Could we create an issue for this, rather than leaving a TODO here that will probably get ignored?

Julesssss replied:
Oh I see, you'll hold the PR on this instead?

Julesssss commented on src/pages/ReimbursementAccount/ExistingOwnersStep.js:
```diff
@@ -0,0 +1,80 @@
+import colors from '../../styles/colors';
+
+const propTypes = {
+    ...withLocalizePropTypes,
```
I'm not sure if this is a rule or not, but don't we usually put these beneath the defined prop types?

marcaaron replied:
No rule that I know of, but that seems to happen more often than not

parasharrajat committed:
fix: scrolling issues

marcaaron reviewed (changes requested):
Before we continue here I'd like to confirm with @Expensify/design that this is what we had in mind.

On the code / organization side of things... I don't think "existing owners" is a big enough thing to be a step. It's not really a step and is just a minor error message.

marcaaron commented on src/pages/ReimbursementAccount/ReimbursementAccountPage.js:
```diff
@@ -232,6 +233,16 @@ class ReimbursementAccountPage extends React.Component {
+                        />
+                    )}
+                    {currentStep === CONST.BANK_ACCOUNT.STEP.EXISTING_OWNERS && (
+                        <ExistingOwnersStep achData={achData} />
```
Steps have a pretty specific significance and are tied to the backend. If we just want to show an error inline here then we can do something similar to the lines above where we are setting the `errorComponent` if there's a problem we need to handle.

rdjuric committed:
removes empty line

Co-authored-by: Rocio Perez <pecanoro@users.noreply.github.com>

parasharrajat committed:
update E-common reference

marcaaron committed:
Merge pull request #3967 from Expensify/marcaaron-navigateToValidationStep

Add link to Concierge on Validation screen when bank account is verifying

parasharrajat committed:
updated e-common & fix bug

parasharrajat committed:
Merge branch 'main' of github.com:Expensify/Expensify.cash into paste-html

pranshuchittora committed:
fix: Minor changes

marcaaron committed:
fix conflict

marcaaron committed:
Merge branch 'marcaaron-getReportSummaryList' of github.com:Expensify/Expensify.cash into marcaaron-getReportSummaryList

pranshuchittora committed:
Merge remote-tracking branch 'origin/main' into pranshuchittora/feat/landing-design

pranshuchittora committed:
fix: Minor string changes

pranshuchittora committed:
chore: Minor string changes

pranshuchittora committed:
fix: Minor string changes

Commit:
Merge branch 'main' of https://github.com/Expensify/Expensify.cash into shawn-updateAppIcon

mountiny committed:
Testing Ecash issue lifecycle

pranshuchittora committed:
fix: File refs

pranshuchittora committed:
fix: Minor terms styling changes

Luke9389 committed:
Merge pull request #3864 from rdjuric/fixTooltipAndDatePos

Fix: Tooltip and message date position

mountiny committed:
Merge pull request #4013 from Expensify/vitTestingECashIssueLifecycle3

pecanoro committed:
Merge pull request #4004 from rdjuric/fixWrongReportOnLHN

Fix wrong active report on LHN

pranshuchittora committed:
fix: adhoc string changes

pranshuchittora committed:
fix: Minor string changes

tgolen committed:
Merge pull request #3988 from parasharrajat/scrollview

fixed scrolling and layout

rafecolton committed:
Change expensify.cash => new.expensify.com

rafecolton committed:
Missed this stuff

mountiny committed:
Testing Ecash issue lifecycle

rdjuric committed:
adds callbackIfInacessible to fetchChatReportsById

rdjuric committed:
fixes comment

jsamr committed:
fix: suppress distracting log box warnings

fix #4014

rdjuric committed:
changes callback to direct redirect in the fetch

roryabraham committed:
Fix display of report typing indicator and participant local time

mountiny committed:
Merge pull request #4017 from Expensify/vitTestingECashIssueLifecycle4

[TEST] - Test E.cash issue lifecycle webhook for employees 3

mountiny committed:
Testing Ecash issue lifecycle

mountiny committed:
Merge pull request #4024 from Expensify/vitTestingECashIssueLifecycle5

[TEST] - Test E.cash issue lifecycle webhook for employees 5

rushatgabhane committed:
remove redundant event listeners from BaseModal

rushatgabhane committed:
add pressOnEnter to replicate prev behaviour of ConfirmModal

rushatgabhane committed:
pressOnEnter for AttachmentModal

Jag96 committed:
Merge pull request #3079 from pranshuchittora/pranshuchittora/feat/landing-design

feat: New landing page design

rushatgabhane committed:
remove redundant onSubmitEditing

rushatgabhane committed:
pressOnEnter for workspaceInvitePage

parasharrajat committed:
refactor

parasharrajat committed:
fix parsing on edit

parasharrajat committed:
update e-common

Jag96 committed:
Merge pull request #4009 from parasharrajat/paste-html

[Feature] Support to html pasting on web

madmax330 committed:
Merge pull request #3873 from Expensify/joel_tierName

Using new parameter name

Beamanator committed:
Merge pull request #4020 from Expensify/Rory-FixUserTypingIndicator

Fix display of report typing indicator and participant local time

Beamanator committed:
Merge pull request #4020 from Expensify/Rory-FixUserTypingIndicator

Fix display of report typing indicator and participant local time

(cherry picked from commit 0639c6771204ceb1c91926813fb2b3368c49a8b6)

Commit:
Update version to 1.0.77-3

OSBotify committed:
Merge pull request #4032 from Expensify/version-BUILD-0639c6771204ceb1c91926813fb2b3368c49a8b6

OSBotify committed:
Merge pull request #4032 from Expensify/version-BUILD-0639c6771204ceb1c91926813fb2b3368c49a8b6

(cherry picked from commit 6cda0beb3272fd0ad51a7641ce89823f4d735687)

rdjuric committed:
navigate to concierge, translated growl

rdjuric committed:
lint fixes and small improvements

tugbadogan committed:
Hide Add Payment Method button

Hide Paypal.me button if there is a Paypal.me username.
Hide Add Payment Method button if there is no available payment method.

tugbadogan committed:
Make Paypal.me user name editable

johnmlee101 committed:
Merge pull request #3925 from Expensify/joe-pin-owed-balances

Treat IOU debts as pinned reports in the LHN

mountiny committed:
Merge pull request #4031 from jsamr/anxiety-spike

fix: suppress distracting log box warnings

Commit:
Remove default state AK and require users to enter a state

roryabraham committed:
Manually resolve merge conflicts

roryabraham committed:
Merge pull request #4033 from Expensify/cherry-pick-staging-4020

🍒 Cherry pick PR #4020 to staging 🍒

Luke9389 committed:
Merge pull request #2635 from Expensify/shawn-updateAppIcon

Update app icons for NewDot branding

Commit:
Add state missing error to be more specific

marcaaron committed:
Merge pull request #3894 from Expensify/marcaaron-getReportSummaryList

Get&returnValueList=reportSummaryList

rdjuric committed:
fetch only if local chat has no reportID

rdjuric committed:
lint

rafecolton committed:
Merge pull request #4016 from Expensify/new-expensify-com

[No QA] Change expensify.cash => new.expensify.com

rafecolton committed:
Merge pull request #4016 from Expensify/new-expensify-com

[No QA] Change expensify.cash => new.expensify.com

(cherry picked from commit 9d900a89ac676dd1ec3b21c2c6d11d287570138b)

rafecolton committed:
Fix incorect staging url

Commit:
Update version to 1.0.77-4

OSBotify committed:
Merge pull request #4038 from Expensify/version-BUILD-9d900a89ac676dd1ec3b21c2c6d11d287570138b

OSBotify committed:
Merge pull request #4038 from Expensify/version-BUILD-9d900a89ac676dd1ec3b21c2c6d11d287570138b

(cherry picked from commit a71b39aead113c75ca55943338a9c2e7dc14a859)

OSBotify committed:
Merge pull request #4039 from Expensify/cherry-pick-staging-4016

roryabraham committed:
Merge pull request #4037 from Expensify/follow-up

[No QA] Fix incorect staging url

roryabraham committed:
Merge pull request #4037 from Expensify/follow-up

[No QA] Fix incorect staging url

(cherry picked from commit 9da2ab6a4f066fe3b77459bf7e4d31458ba1f77f)

Commit:
Update version to 1.0.77-5

OSBotify committed:
Merge pull request #4041 from Expensify/version-BUILD-9da2ab6a4f066fe3b77459bf7e4d31458ba1f77f

OSBotify committed:
Merge pull request #4041 from Expensify/version-BUILD-9da2ab6a4f066fe3b77459bf7e4d31458ba1f77f

(cherry picked from commit 04ba0ec62afd2245ee8002b9d883ef96f58a8b86)

OSBotify committed:
Merge pull request #4042 from Expensify/cherry-pick-staging-4037

Commit:
Update version to 1.0.77-6

OSBotify committed:
Merge pull request #4044 from Expensify/version-BUILD-04ba0ec62afd2245ee8002b9d883ef96f58a8b86

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #4045 from Expensify/update-staging-from-main

parasharrajat committed:
fix: action failure on invalid String in BASH

Commit:
changed app store icon

johncschuster committed:
Merge branch 'main' into JohnSchusterECashDocEdits

AndrewGable committed:
Merge pull request #4048 from Expensify/luke-new-app-store-icon

[No QA] Changed App Store Icon

Commit:
Update version to 1.0.77-7

OSBotify committed:
Merge pull request #4050 from Expensify/version-BUILD-37d013e3e55bcfafbf47fb534cd9d7c7a67b5947

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #4051 from Expensify/update-staging-from-main

Commit:
Update version to 1.0.78-0

OSBotify committed:
Merge pull request #4053 from Expensify/version-PATCH-c1c29bc863dd907553b6aabc08c8be91c6477de8

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #4054 from Expensify/update-staging-from-main

tugbadogan committed:
Fixed API call

rdjuric committed:
resolution for conflict

rdjuric committed:
lint fixes

rdjuric committed:
minor changes

Jag96 committed:
Don't show all chats in LHN in #focus mode

Santhosh-Sellavel committed:
Restrict self requests & naming convention changes

marcaaron committed:
Merge pull request #4059 from Expensify/joe-report-action-count

[CP Staging] Don't show all chats in LHN in #focus mode

(cherry picked from commit 951308fb3319f64c10b3302f904b26b65d72801e)

marcaaron committed:
Merge pull request #4059 from Expensify/joe-report-action-count

[CP Staging] Don't show all chats in LHN in #focus mode

Commit:
Update version to 1.0.78-1

OSBotify committed:
Merge pull request #4064 from Expensify/version-BUILD-951308fb3319f64c10b3302f904b26b65d72801e

OSBotify committed:
Merge pull request #4064 from Expensify/version-BUILD-951308fb3319f64c10b3302f904b26b65d72801e

(cherry picked from commit 1a1d66a322c733bbc79f8a5f05f1527240c769d7)

OSBotify committed:
Merge pull request #4065 from Expensify/cherry-pick-staging-4059

parasharrajat committed:
update e-common

Beamanator committed:
Update about page copy

Beamanator committed:
Fix translation of "New" to "El nuevo"

deetergp committed:
Merge pull request #3992 from Expensify/Rory-DisabledButtonStyle

Give disabled buttons the disabled cursor style

sketchydroide committed:
Merge pull request #4069 from Expensify/beaman-updateAboutPageWithNewExpensifyText

Update about page copy

roryabraham committed:
Merge pull request #4034 from tugbadogan/tugbadogan-hide-add-payment-button

Hide add payment button

johncschuster committed:
Merge pull request #3984 from Expensify/JohnSchusterECashDocEdits

[Hold - To be deployed with new.expensify.com changes] Scrubbing Expensify Cash for New Expensify

parasharrajat committed:
fix text selection on resized web devices

iwiznia committed:
Remove unnecessary calls to updateLastRead

pecanoro committed:
Merge pull request #4035 from Expensify/amal-remove-default-state-pick

Remove default state AK and require users to enter a state

Commit:
Merge pull request #4078 from Expensify/ionatan_unneceesary_lastread

Remove unnecessary calls to updateLastRead

Commit:
Merge pull request #4063 from Santhosh-Sellavel/santhosh_self_split_bill_and_request

Restrict self requests & naming convention changes

rushatgabhane committed:
merge main, fix conflict in Button

rushatgabhane committed:
fix typo

iwiznia committed:
Merge pull request #4079 from parasharrajat/fix-blue

chiragsalian committed:
Reverting 3979

Commit:
Load policies after we load betas as well

Commit:
Some DRY for repeated code

chiragsalian committed:
Merge pull request #4085 from Expensify/chirag-revert-3979

Reverting 3979

(cherry picked from commit d15c65a3095fe166e458181ed9e14d3b0adc4e56)

chiragsalian committed:
Merge pull request #4085 from Expensify/chirag-revert-3979

Reverting 3979

Commit:
Update version to 1.0.78-2

OSBotify committed:
Merge pull request #4087 from Expensify/version-BUILD-d15c65a3095fe166e458181ed9e14d3b0adc4e56

(cherry picked from commit b6d2c0de0d34495db15910ceb61f5ac2ae0ec03e)

OSBotify committed:
Merge pull request #4087 from Expensify/version-BUILD-d15c65a3095fe166e458181ed9e14d3b0adc4e56

OSBotify committed:
Merge pull request #4088 from Expensify/cherry-pick-staging-4085

luacmartins committed:
Merge pull request #3977 from rushatgabhane/enter-go-next

Web/Desktop Press Enter key to go next

tgolen committed:
Merge pull request #4084 from Expensify/amal-load-policy-beta

Load policies after we load betas as well

aliabbasmalik8 committed:
[ISSUE-4086] Added initial number in FlatList

pecanoro committed:
Merge pull request #4089 from aliabbasmalik8/ISSUE-4086-add-initial-number

[ISSUE-4086] Added initial number in FlatList

Commit:
Update version to 1.0.78-3

OSBotify committed:
Merge pull request #4092 from Expensify/version-BUILD-219daeb79a6cb3922992beaaf8a5356ca1229901

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #4093 from Expensify/update-staging-from-main

Jag96 committed:
Merge pull request #4067 from parasharrajat/paste-html

Fix Line-break parsing issue in E.cash.

Commit:
Update version to 1.0.78-4

OSBotify committed:
Merge pull request #4094 from Expensify/version-BUILD-f8de6ee9f753ebde90d87ee74a1303ae63b15d1e

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #4095 from Expensify/update-staging-from-main

roryabraham committed:
Make a dummy change for a test PR

roryabraham committed:
Merge pull request #4096 from Expensify/Rory-DummyPR

[No QA] Make a dummy change for a test PR

roryabraham committed:
Undo dummy change

roryabraham committed:
Merge pull request #4098 from Expensify/Rory-UndoDummyPR

[No QA] Undo dummy change

Commit:
Update version to 1.0.78-5

OSBotify committed:
Merge pull request #4099 from Expensify/version-BUILD-d0671027d4a09ab72974c16ffa573c9459b03f2c

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #4100 from Expensify/update-staging-from-main

Commit:
Update version to 1.0.79-0

OSBotify committed:
Merge pull request #4102 from Expensify/version-PATCH-5c5687bb00699bfa761b1c560259c7e6d0305160

Commit:
Merge branch 'main' into update-staging-from-main

OSBotify committed:
Merge pull request #4103 from Expensify/update-staging-from-main

mountiny committed:
Merge pull request #4081 from parasharrajat/animated-2

[No QA] Configured Reanimated 2 for Drawer

roryabraham committed:
Merge pull request #4113 from Expensify/staging

[No QA] Update staging from main

tgolen committed:
Remove duplicate form fields

roryabraham committed:
Merge pull request #4117 from Expensify/tgolen-fix-fields

[CP Staging] Remove duplicate form fields

Commit:
Update version to 1.0.79-1

OSBotify committed:
Merge pull request #4124 from Expensify/version-BUILD-92fd456846907b28ef5270d6d284ce1deceb98b8

parasharrajat committed:
fix: paste and Ctrl+z

parasharrajat committed:
Merge branch 'main' of github.com:Expensify/Expensify.cash into paste-html

parasharrajat committed:
remove extra selection

HorusGoul committed:
Merge pull request #4131 from parasharrajat/paste-html

fix: paste and Ctrl+z

tgolen committed:
Merge pull request #4049 from parasharrajat/actions-deployblock

[NO QA] fix: action failure on invalid String in BASH

marcaaron committed:
Stop call to Get=reportStuff with reportID = undefined

rdjuric committed:
updates to Report and comment to ReportActionsView

rdjuric committed:
lint fixes

marcaaron committed:
use onLayout instead of debouncing report action items

marcaaron committed:
Fix timing for chat switching by using onLayout

Commit:
Don't check participants' whole names in default rooms

Commit:
Update podfile.lock checksum

marcaaron committed:
Merge pull request #4018 from rdjuric/navigatingToInacessibleChat

Show a growl and redirect when navigating to a chat we can't access

Luke9389 committed:
Merge pull request #4139 from Expensify/marcaaron-reportStuff

Stop call to Get&rvl=reportStuff with reportID = undefined

yuwenmemon committed:
Modify line height so text does not get cut off in LHN

yuwenmemon committed:
adjust line height of option display name to avoid cutoff in Android

yuwenmemon committed:
Use round avatar for E.Cash room avatar

yuwenmemon committed:
remove background color from avatars

mountiny committed:
RenameGetPreferredCurrency API command to GetLocalCurrency

iwiznia committed:
Merge pull request #4140 from Expensify/marcaaron-useOnLayout

Julesssss committed:
Merge pull request #4145 from Expensify/vit-fixAPICommand

[No QA] RenameGetPreferredCurrency API command to GetLocalCurrency

Commit:
Merge pull request #4144 from Expensify/yuwen-roundRoom

Use round avatar for E.Cash room avatar

Commit:
Update version to 1.0.79-2

OSBotify committed:
Merge pull request #4148 from Expensify/version-BUILD-7b5d3edba6087d6d2863502c59104cd2b24b9bcb

thienlnam committed:
Merge pull request #4141 from Expensify/amal-participants-full-name-check

Don't check participants' whole names in default rooms

yuwenmemon committed:
Merge pull request #4143 from Expensify/yuwen-lineHeight

Modify line height so text does not get cut off in LHN

Commit:
Update version to 1.0.79-3

OSBotify committed:
Merge pull request #4151 from Expensify/version-BUILD-029ed175ecaaed0704bada0de67c6b9a24b51844

jasperhuangg committed:
Display error in confirm modal instead of step

jasperhuangg committed:
Merge remote-tracking branch 'origin/jasper-existingOwnersCopy' into jasper-existingOwnersCopy

# Conflicts:
#	src/libs/actions/BankAccounts.js

jasperhuangg committed:
remove debug

jasperhuangg commented:
> **CLA Assistant Lite bot:**
> Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you all sign our [Contributor License Agreement](https://github.com/Expensify/App/blob/main/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> _**I have read the CLA Document and I hereby sign the CLA**_
> 
> **16** out of **18** committers have signed the CLA.
> ✅ @jasperhuangg
> ✅ @rdjuric
> ✅ @roryabraham
> ✅ @marcaaron
> ✅ @parasharrajat
> ✅ @rushatgabhane
> ✅ @johncschuster
> ✅ @Luke9389
> ✅ @AndrewGable
> ✅ @thienlnam
> ✅ @mountiny
> ✅ @vitHoracek
> ✅ @pranshuchittora
> ✅ @tgolen
> ✅ @rafecolton
> ✅ @Jag96
> ❌ @madmax330
> ❌ @pecanoro
> You can retrigger this bot by commenting **recheck** in this Pull Request

Sorry for all the mentions! Something happened when I was resolving a merge that led to this branch getting messed up. Please ignore this 😓
